### ISSUE TITLE
Add constraint statement syntax error?

### ISSUE THREAD

**Issue by kenny-shen** [sbdchd/squawk#479]
The following statement results in syntax errors under Squawk 2.6.0,

```sql
alter table foo
  add constraint foo_bar_id_fkey foreign key (bar_id)
  references bar (id) on update cascade on delete cascade;
```

```
warning[syntax-error]: expected DELETE_KW
 --> foo.sql:3:25
  |
3 |   references bar (id) on update cascade on delete cascade;
  |                         ^
  |
warning[syntax-error]: expected SEMICOLON
 --> foo.sql:3:25
  |
3 |   references bar (id) on update cascade on delete cascade;
  |                         ^
  |
warning[syntax-error]: expected SET_KW
 --> foo.sql:3:40
  |
3 |   references bar (id) on update cascade on delete cascade;
  |                                        ^
  |
warning[syntax-error]: expected name
 --> foo.sql:3:40
  |
3 |   references bar (id) on update cascade on delete cascade;
  |                                        ^
  |
warning[syntax-error]: expected EQ
 --> foo.sql:3:40
  |
3 |   references bar (id) on update cascade on delete cascade;
  |                                        ^
  |
warning[syntax-error]: expected an expression, found ON_KW
 --> foo.sql:3:41
  |
3 |   references bar (id) on update cascade on delete cascade;
  |                                         ^
  |
warning[syntax-error]: expected expression
 --> foo.sql:3:43
  |
3 |   references bar (id) on update cascade on delete cascade;
  |                                           ^
  |
warning[syntax-error]: expected SEMICOLON
 --> foo.sql:3:43
  |
3 |   references bar (id) on update cascade on delete cascade;
  |                                           ^
  |
warning[syntax-error]: expected FROM_KW
 --> foo.sql:3:50
  |
3 |   references bar (id) on update cascade on delete cascade;
  |                                                  ^
  |
warning[constraint-missing-not-valid]: By default new constraints require a table scan and block writes to the table while that scan occurs.
 --> foo.sql:2:3
  |
2 | /   add constraint foo_bar_id_fkey foreign key (bar_id)
3 | |   references bar (id) on update cascade on delete cascade;
  | |________________________^
  |
  = help: Use `NOT VALID` with a later `VALIDATE CONSTRAINT` call.
warning[adding-foreign-key-constraint]: Adding a foreign key constraint requires a table scan and a `SHARE ROW EXCLUSIVE` lock on both tables, which blocks writes to each table.
 --> foo.sql:2:7
  |
2 |     add constraint foo_bar_id_fkey foreign key (bar_id)
  |  _______^
3 | |   references bar (id) on update cascade on delete cascade;
  | |________________________^
  |
  = help: Add `NOT VALID` to the constraint in one transaction and then VALIDATE the constraint in a separate transaction.

Find detailed examples and solutions for each rule at https://squawkhq.com/docs/rules
Found 11 issues in 1 file (checked 1 source file)
```

**Comment by sbdchd** [comment]
Thank you for the report, fixed in version v2.7.0. Builds should be up shortly!
